Applications/Terminal/Terminal.cpp

```diff
@@ -389,18 +446,22 @@ void Terminal::escape$J(const ParamVector& params)
     switch (mode) {
     case 0:
         // Clear from cursor to end of screen.
-        for (int i = m_cursor_column; i < m_columns; ++i) {
+        for (int i = m_cursor_column; i < m_columns; ++i)
             put_character_at(m_cursor_row, i, ' ');
-        }
         for (int row = m_cursor_row + 1; row < m_rows; ++row) {
             for (int column = 0; column < m_columns; ++column) {
                 put_character_at(row, column, ' ');
             }
         }
         break;
     case 1:
-        // FIXME: Clear from cursor to beginning of screen.
-        unimplemented_escape();
+        for (int i = m_cursor_column-1; i >= 0; --i)
```

Comment:
I'd suggest leaving the comment in place:

    // Clear from the beginning of the display to the cursor